packages/rest/src/rest.server.ts

```diff
@@ -670,6 +708,15 @@ export interface RestServerOptions {
   cors?: cors.CorsOptions;
   apiExplorerUrl?: string;
   sequence?: Constructor<SequenceHandler>;
+  /**
+   * Static assets to be served using `express.static()`
```

Comment:
See https://github.com/strongloop/loopback-next/issues/742.

> The ApplicationConfig class should contain only things like HTTP port and database connection strings, there should be no entries related to what component, controller, repositories (etc.) the application is using. User application classes should call this.controller(), this.repository() (etc.) to register their artifacts, and eventually use the new bootstrapper module.

Please remove this config option in favor of `app.static` API.